### ISSUE TITLE
Sidekiq Enterprise collaboration

### ISSUE THREAD

**Issue by reedloden** [mperham/sidekiq#4060]
As per https://github.com/mperham/sidekiq/wiki/Commercial-collaboration, I am requesting access to the private Sidekiq Enterprise repository.

E-mail address used for purchase: it-purchasing [at] hackerone.com

I assign all rights, including copyright, to any future Sidekiq Enterprise work by myself to Mike Perham.

**Comment by mperham** [comment]
Do you want Pro access too?

**Comment by reedloden** [comment]
Yes, but I already have Pro access, I believe. Please double check. :-)
